vizro-core/examples/_dev/app.py

```diff
@@ -5,26 +5,41 @@
 from vizro import Vizro
 from vizro.tables import dash_ag_grid, dash_data_table
 
-df = px.data.gapminder()
+NUMBER_OF_COMPONENTS = 64
```

Comment:
Not sure if this is related to the large number of components, but when I run the dev example and change the themes it gives me a bunch of `id` errors 🤔  Could you double-check?

![Screenshot 2024-06-13 at 13 57 28](https://github.com/mckinsey/vizro/assets/90609403/3a2d395e-4b8f-4975-a1bf-a2f41f62e117)

Reply:
Ahh good catch. Does it work now?

Reply:
For me yes!

Reply:
FYI: The following change (in the `Graph.build clientside_callback`) makes it to work properly now 😄 
```python
            # Output here to ensure that the callback is only triggered if the graph exists on the currently open page.
            output=[Output(self.id, "figure")],
```